server/container_create_linux.go

```diff
@@ -215,10 +215,7 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 	}
 
 	// creates a spec Generator with the default spec.
```

Comment:
nit (fine as follow up) we don't actually create the spec here, so the comment is out of date

Reply:
Fixed the comment :+1: